node/cmd/guardiand/node.go

```diff
@@ -989,165 +909,270 @@ func runNode(cmd *cobra.Command, args []string) {
 				near.NewWatcher(*nearRPC, *nearContract, lockC, chainObsvReqC[vaa.ChainIDNear], !(*unsafeDevMode || *testnetMode)).Run); err != nil {
 				return err
 			}
 		}
 
 		// Start Wormchain watcher only if configured
-		if *wormchainWS != "" && *wormchainLCD != "" {
+		if *wormchainWS != "" && *wormchainLCD != "" && *wormchainWS != "none" {
 			logger.Info("Starting Wormchain watcher")
+			readiness.RegisterComponent(common.ReadinessWormchainSyncing)
+			chainObsvReqC[vaa.ChainIDWormchain] = make(chan *gossipv1.ObservationRequest, observationRequestBufferSize)
 			if err := supervisor.Run(ctx, "wormchainwatch",
 				wormchain.NewWatcher(*wormchainWS, *wormchainLCD, lockC, setC, chainObsvReqC[vaa.ChainIDWormchain]).Run); err != nil {
 				return err
 			}
 		}
-		if *aptosRPC != "" {
+		if *aptosRPC != "" && *aptosRPC != "none" {
+			logger.Info("Starting Aptos watcher")
+			readiness.RegisterComponent(common.ReadinessAptosSyncing)
+			chainObsvReqC[vaa.ChainIDAptos] = make(chan *gossipv1.ObservationRequest, observationRequestBufferSize)
 			if err := supervisor.Run(ctx, "aptoswatch",
 				aptos.NewWatcher(*aptosRPC, *aptosAccount, *aptosHandle, lockC, chainObsvReqC[vaa.ChainIDAptos]).Run); err != nil {
 				return err
 			}
 		}
 
-		if *solanaRPC != "" {
+		if *solanaRPC != "" && *solanaRPC != "none" {
+			logger.Info("Starting Solana watcher")
+			readiness.RegisterComponent(common.ReadinessSolanaSyncing)
+			chainObsvReqC[vaa.ChainIDSolana] = make(chan *gossipv1.ObservationRequest, observationRequestBufferSize)
 			if err := supervisor.Run(ctx, "solwatch-confirmed",
 				solana.NewSolanaWatcher(*solanaRPC, solAddress, lockC, nil, rpc.CommitmentConfirmed, common.ReadinessSolanaSyncing, vaa.ChainIDSolana).Run); err != nil {
 				return err
 			}
-
 			if err := supervisor.Run(ctx, "solwatch-finalized",
 				solana.NewSolanaWatcher(*solanaRPC, solAddress, lockC, chainObsvReqC[vaa.ChainIDSolana], rpc.CommitmentFinalized, common.ReadinessSolanaSyncing, vaa.ChainIDSolana).Run); err != nil {
 				return err
 			}
 		}
 
-		if *pythnetRPC != "" {
+		if *pythnetRPC != "" && *pythnetRPC != "none" {
+			logger.Info("Starting Pythnet watcher")
+			readiness.RegisterComponent(common.ReadinessPythNetSyncing)
+			chainObsvReqC[vaa.ChainIDPythNet] = make(chan *gossipv1.ObservationRequest, observationRequestBufferSize)
 			if err := supervisor.Run(ctx, "pythwatch-confirmed",
 				solana.NewSolanaWatcher(*pythnetRPC, pythnetAddress, lockC, nil, rpc.CommitmentConfirmed, common.ReadinessPythNetSyncing, vaa.ChainIDPythNet).Run); err != nil {
 				return err
 			}
-
 			if err := supervisor.Run(ctx, "pythwatch-finalized",
 				solana.NewSolanaWatcher(*pythnetRPC, pythnetAddress, lockC, chainObsvReqC[vaa.ChainIDPythNet], rpc.CommitmentFinalized, common.ReadinessPythNetSyncing, vaa.ChainIDPythNet).Run); err != nil {
 				return err
 			}
 		}
 
+		if *testnetMode {
+			if *ethRopstenRPC != "" {
+				logger.Info("Starting Eth Ropsten watcher")
```

Comment:
Could we also add the != "none" here?  That way, when we bring them live we just have to move them instead of having to change them

Reply:
Done

Reply:
Done